huaweicloud/resource_huaweicloud_vpc_test.go

```diff
@@ -167,3 +226,18 @@ resource "huaweicloud_vpc" "test" {
 }
 `, rName, OS_ENTERPRISE_PROJECT_ID)
 }
+
+func tesstAccVpcV1_WithCustomRegion(name1 string, name2 string, region string) string {
+	return fmt.Sprintf(`
+resource "huaweicloud_vpc" "test1" {
+  name = "%s"
+  cidr = "192.168.0.0/16"
+}
+
+resource "huaweicloud_vpc" "test2" {    
+	name = "%s"
+	region = "%s"
+    cidr = "192.168.0.0/16"
+}
```

Comment:
please format the resource with [HCL style](https://www.terraform.io/docs/configuration/style.html)

Reply:
I have fixed this by modifing it’s indentation into two blanks.